main.go

```diff
@@ -102,6 +103,7 @@ var modules = map[string](func(*powerline)){
 	"time":     segmentTime,
 	"user":     segmentUser,
 	"venv":     segmentVirtualEnv,
+	"shell-var": segmentShellVar,
```

Comment:
This list should be alphabetically sorted, and properly indented

Reply:
The module also needs to be added to the section here: https://github.com/mexchip/powerline-go/blob/eda2d5c5efae958a2efa142870cbd1b57f085d42/main.go#L163